src/ops/embedding.cc

```diff
@@ -90,21 +91,18 @@ EmbeddingParams Embedding::get_params() const {
   params.num_entries = this->num_entries;
   params.out_channels = this->out_channels;
   params.aggr = this->aggr;
-
+  params.layer_guid = this->layer_guid;
   return params;
 }
 
-size_t EmbeddingParams::get_hash(const ParallelTensor input) const {
-  size_t hash = input->get_owner_independent_hash();
-  hash_combine(hash, this->num_entries);
-  hash_combine(hash, this->out_channels);
-  hash_combine(hash, this->aggr);
-
-  return hash;
+bool EmbeddingParams::is_valid(ParallelTensorShape const &input) const {
+  return input.is_valid();
```

Comment:
Should there be a `TODO` here? Is this really the only constraint for `Embedding`?

Reply:
Based on [this](https://github.com/flexflow/FlexFlow/blob/master/src/ops/embedding.cc#L715), it does not seem like there are other constraints.